### ISSUE TITLE
[Feature request]: create a function based on psed method to project future spread event days

### ISSUE THREAD

**Issue by denysed** [BadgerOnABike/BurnP3.HelpR#9]
Hey Badger- I did some testing of Wang et al. 2014/ 2017's psed (with link function) for projecting spread vs seeing if the sp_event_day function could be used for this purpose (just want to put out there- I know the sp_event_day fx as written isn't necessarily intended for this).

The Wang et al. method produces more realistic spread _projections_. But, the method requires users to already have a validated "realized spread event day" distribution.

I was thinking/wondering if a possible workflow here could be (?):
1. the sp_event_day function gets used by the majority of users to produce their initial spread event day distributions, if they don't have MODIS hot spot data or another way to produce that distribution. 
2. Those distributions get validated based on historical data, as we do.
3.  If the user is projecting their data into the future, they could use a function that wraps all of the Wang et al. method to project that validated SED dist produced in steps 1 & 2 to get future SED distributions.

Basically, I'm wondering if it would be useful to eventually develop a suite of functions to help users project BP3 inputs to use for analyses of future conditions. This could be the method and workflow for users to project their SED distributions. Thoughts?

**Comment by BadgerOnABike** [comment]
Yup, that's exactly the plan.

Due to the nature of the projects I've been working on I've had to use potential sed. The ideal flow is to check for MODIS spots first then move back to weather. Blending from reality into projection is a long term step as it'll likely take a fair bit of discussion and thinking on how to do this reasonably.

**Comment by kdbeatti** [comment]
Hey badger.

Big fan of your work here!
Thanks for sharing all this.  Solid Gold. Did anything come of this? If there is still interest I might give this module the old college try.....

**Comment by BadgerOnABike** [comment]
I haven't pushed in on this yet, got sucked into a bunch of work with mountain pine beetle. Definitely still interest as this will likely be ingested in a number of folks workflows! Have at it, I'm happy to look at stuff as you go too if you hit any roadblocks!

**Comment by kdbeatti** [comment]
Ok, thanks! I will definitely take you up on looking at stuff as I hit roadblocks....just started using R so it won't be as polished. Did you use these for the parks burn-p3 project?

**Comment by BadgerOnABike** [comment]
I did indeed! The reup stuff at least. The initial full run was done prior to having this established.
